tools/palace-sync.ps1

```
@@ -0,0 +1,266 @@
+$ProposalLines.Add("")
+$ProposalLines.Add("---")
+$ProposalLines.Add("")
+
+# --- Parse commits ---
+$RoomGroups = @{}
+$CurrentHash = ""
+$CurrentDate = ""
+$CurrentMsg  = ""
+$CurrentFiles = [System.Collections.Generic.List[string]]::new()
+
+function Process-Commit {
```

PSScriptAnalyzer (1.25.0)

[warning] The cmdlet 'Process-Commit' uses an unapproved verb.

(PSUseApprovedVerbs)